src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,56 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readDataFromFile(fromFileName);
+        writeDataToFile(toFileName, totalStatCalculation(readDataFromFile(fromFileName)));
+    }
+
+    private String[] readDataFromFile(String fromFileName) {
+        StringBuilder builder = new StringBuilder();
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String currentLine = reader.readLine();
+            while (currentLine != null) {
+                builder.append(currentLine).append(System.lineSeparator());
+                currentLine = reader.readLine();
+            }
+        } catch (Exception e) {
+            throw new RuntimeException("Can't read file: " + fromFileName, e);
+        }
+        return builder.toString().split(System.lineSeparator());
+    }
+
+    private String totalStatCalculation(String[] splitStrings) {
```

Comment:
```suggestion
    private String calculateResult(String[] splitStrings) {
```
use verbs for method names